### PR TITLE
Fix new border, oonodz logo, spacing

### PR DESCRIPTION
**Pull request by emersoncloud**

### CODE REVIEW

File: src/hooks/useMinipoolStreamliner.ts [hunk 1]
```diff
@@ -68,6 +71,18 @@ export const useCreateStreamlinedMinipool = (
...
+      const stakingEventTopic = new ethers.utils.Interface(stakingAbi).getEventTopic('GGPStaked')
+      let ggpStaked = BigNumber.from(0)
+      const stakingEvent = receipt.logs.find((log) => log.topics[0] === stakingEventTopic)
+      if (stakingEvent && stakingEvent.data) {
+        const stakingDecodedEvent = new ethers.utils.Interface(stakingAbi).decodeEventLog(
+          'GGPStaked',
+          stakingEvent.data,
+        )
+        ggpStaked = stakingDecodedEvent?.amount
+      }
```
Comment (jacktrusler) [inline src/hooks/useMinipoolStreamliner.ts:84]: Nice, this shows the amount of ggp staked by looking through the logs to find the AVAX -> GGP conversion?

File: src/modules/components/MinipoolQuickstart/MQSuccess.tsx [hunk 2]
```diff
@@ -108,6 +114,7 @@ const MQSuccess = ({ formData, transactionData }) => {
             width="100%"
           >
             <InfoBox label="AVAX DEPOSITED" value={displayBN(formData.deposit)} />
+            <InfoBox label="GGP BOUGHT" value={displayBN(transactionData.ggpStaked)} />
```
Comment (jacktrusler) [inline src/modules/components/MinipoolQuickstart/MQSuccess.tsx:117]: nice touch 👌

File: src/modules/components/MinipoolQuickstart/inputs/ValidationLengthInput.tsx [hunk 3]
```diff
@@ -33,7 +34,7 @@ export default function ValidationLengthInput({ formData, setFormData }: Validat
           </Text>
         </span>
         <select
-          className="w-full truncate border-none pl-1 focus:border-none focus:outline-none"
+          className="w-full appearance-none truncate border-none pl-1 focus:border-none focus:outline-none"
```
Comment (jacktrusler) [inline src/modules/components/MinipoolQuickstart/inputs/ValidationLengthInput.tsx:37]: safari is cringe for this

File: src/hooks/useMinipoolStreamliner.ts (hunk 1 above)
Comment (emersoncloud) [reply src/hooks/useMinipoolStreamliner.ts:84]: that's right. event logs, just like we do for node id

**Pull request merged**